src/components/ui/ListingsHome.tsx

```
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function ListingsHome({ children, title, seeAllLink, className }: { children: React.ReactNode, title: string, seeAllLink: string, className?: string }) {
+    return (
+        <div className={`container ${className}`}>
+            <div className='flex justify-between items-center mb-6'>
+                <p className='font-medium text-2xl'>{title}</p>
+                <button className='font-normal text-lg text-neutral' onClick={() => window.location.href = seeAllLink}>see all</button>  {/* Simple navigation to see all listings //* i will change it for Link */}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)